sql/create_user.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO users (username, token, salt, balance, collected_timestamp)
+VALUES ($1, $2, $3, $4, $5) RETURNING *;
```

Comment:
Nothing big but this is a bit inconsistent
```suggestion
VALUES ($1, $2, $3, $4, $5)
RETURNING *;
```